README.md

```diff
@@ -1 +1,77 @@
-# Module1-Capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Module1-Capstone
+
+> Module-1 capstone project.
+
```

Comment:
- [ ] OPTIONAL: In my opinion project title and description are repeating I suggest to write one or two sentences describing the project. It will help others to understand the project you worked so hard 💪🏻